Makefile

```diff
@@ -12,19 +12,13 @@ help: ## Show all Makefile targets
 .PHONY: format format-proto lint lint-proto type style clean
 format: ## Running code formatter: black and isort
 	@echo "(black) Formatting codebase..."
-	@black --config pyproject.toml src tests docs examples
-	@echo "(black) Formatting stubs..."
-	@find src -name "*.pyi" ! -name "*_pb2*" -exec black --pyi --config pyproject.toml {} \;
-	@echo "(isort) Reordering imports..."
-	@isort .
-	@echo "(ruff) Running fix only..."
-	@ruff check src examples tests --fix-only
+	@pre-commit run --all-files black
```

Comment:
We should keep isort right?

Reply:
Since both ruff and isort sort imports, there will be some conflict when people install one in their dev environment.

Reply:
I think ruff already did all of what isort already doing with `--fix`. +1 for removing isort

Reply:
Ah,  got it!